lib/suma/member.rb

```diff
@@ -104,6 +105,10 @@ def with_email(*emails)
     def with_normalized_phone(*phones)
       return self.where(phone: phones)
     end
+
+    def active
```

Comment:
There is already `not_soft_deleted` which is more explicit. What is 'active'?

Reply:
Good catch, wasn't aware of it.